src/System.ComponentModel.TypeConverter/tests/TypeConverterTests.cs

```diff
@@ -71,25 +72,27 @@ public static void ConvertFromString()
         [Fact]
         public static void ConvertFrom_InstanceDescriptor()
         {
-            RemoteExecutor.Invoke(() =>
+            RemoteExecutorForUap.Invoke(() =>
             {
-                CultureInfo.CurrentCulture = CultureInfo.GetCultureInfo("fr-FR");
-                DateTime testDateAndTime = DateTime.UtcNow;
-                ConstructorInfo ctor = typeof(DateTime).GetConstructor(new Type[]
+                using (new ThreadCultureChange("fr-FR"))
```

Comment:
Did you consider adding culture as a parameter to Invoke? Most of these tests don't need to set the culture back, and you wouldn't need to use ThreadCultureChange.

perhaps that would make too many overloads.

Reply:
I considered it.  It would increase the number of overloads as you say.  It's also orthogonal to the purpose of RemoteExecutor.Invoke.  And when we eventually no longer have a need for RemoteExecutorForUap.Invoke, we can go and delete all of them, but we'll still need the ThreadCultureChange.